tests/ci/cdk/cdk/codebuild/github_ci_analytics_omnibus.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+version: 0.2
+
+# Doc for batch https://docs.aws.amazon.com/codebuild/latest/userguide/batch-build-buildspec.html#build-spec.batch.build-list
+batch:
+  build-list:
+
+    - identifier: ubuntu2004_clang10_x86_64_analytics
+      buildspec: ./tests/ci/codebuild/common/run_analytics.yml
+      env:
+        type: LINUX_CONTAINER
+        privileged-mode: true
+        compute-type: BUILD_GENERAL1_LARGE
+        image: X86_ECR_REPO_PLACEHOLDER:ubuntu-20.04_clang-10x_latest
```

Comment:
Nit: we are only interested in the metrics of build with `clang-10`?

Reply:
For right now yes, do you think it makes sense to add compiler as a dimension to the metrics for the future? CloudWatch has a [limit of 10 dimensions](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/cloudwatch_limits.html) per metric. For the library size metrics we're at 8 (library, release, assembly, small, static/shared, fips, CPU, branch). I can't think of that many more dimensions for size we might be interested in the future. It is making me think that instead of the metric LibrarySize and a dimension of Library (name) the metric could be CryptoSize and SSLSize. It doesn't really make sense to want to search/graph libcrypto and libssl size together with the name as a dimension.